t/25-cache.t

```diff
@@ -199,6 +199,16 @@ like $cache_log,
   'Reclaimed space for new smaller asset';
 $cache_log = '';
 
+$cache->get_asset($host, {id => 922756}, 'hdd', 'sle-12-SP3-x86_64-0368-200_#@64bit.qcow2');
+like $cache_log, qr/Download of ".*sle-12-SP3-x86_64-0368-200_#.*" successful/,
+  'Asset with # was downloaded successfully';
+$cache_log = '';
+
+$cache->get_asset($host, {id => 922756}, 'hdd', 'sle-12-SP3-x86_64-0368-200_:@64bit.qcow2');
+like $cache_log, qr/Download of ".*sle-12-SP3-x86_64-0368-200_:.*" successful/,
+  'Asset with : was downloaded successfully';
+$cache_log = '';
```

Comment:
As the test and you repeatedly delete the `$cache_log` variable I suggest to extract that into a test method but also call the above two tests with a for-loop instead of copy-pasting